src/rqt_plot/data_plot/mat_data_plot.py

```diff
@@ -101,28 +101,14 @@ def __init__(self, parent=None):
             super(MatDataPlot.Canvas, self).__init__(Figure())
             self.axes = self.figure.add_subplot(111)
             self.axes.grid(True, color='gray')
-            self.safe_tight_layout()
+            self.figure.tight_layout()
```

Comment:
We still need `safe_tight_layout()`. This line doesn't work for me. It crashes right away when I type `rqt_plot` with this:
```
$ rqt_plot
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/matplotlib/backends/backend_qt5agg.py", line 77, in paintEvent
    self.resizeEvent(event)
  File "/data/ws/ros/viper/catkin_ws/src/ros-visualization/rqt_plot/src/rqt_plot/data_plot/mat_data_plot.py", line 111, in resizeEvent
    self.figure.tight_layout()
  File "/usr/lib/python2.7/dist-packages/matplotlib/figure.py", line 2031, in tight_layout
    self.subplots_adjust(**kwargs)
  File "/usr/lib/python2.7/dist-packages/matplotlib/figure.py", line 1881, in subplots_adjust
    self.subplotpars.update(*args, **kwargs)
  File "/usr/lib/python2.7/dist-packages/matplotlib/figure.py", line 241, in update
    raise ValueError('bottom cannot be >= top')
ValueError: bottom cannot be >= top
Aborted (core dumped)
```